src/sentry/static/sentry/app/components/customResolutionModal.jsx

```diff
@@ -7,7 +7,7 @@ import {SelectAsyncField} from 'app/components/forms';
 import {t} from 'app/locale';
 import Button from 'app/components/button';
 import TimeSince from 'app/components/timeSince';
-import Version from 'app/components/version';
```

Comment:
is version 1 something we can get rid of?

Reply:
Yes, as a matter of fact, it is not being used anywhere at this point. At first, I thought that we would do this more gradually, that is why I left the v1 in the codebase. I guess there is no point in waiting, so I've updated the PR with removed v1 and renamed v2.